index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="author" content="Hamam ALsamel">
+    <meta name="description" content="I am Telecom Engineer and web developer">
+    <title>
+    <strong>Introducing Myself</strong> </title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+  </head>
+  <body>
+    <header>
+    <h1>Introducing My Myself</h1>
+    <img src="images/0.jpg" alt="profile" class="profile">
+
+
+    <p> My name is <strong>Hamam ALsamel</strong> i am telecome engineer with 5 years experience, I have experience in two diffrent role which is electrical and telecom field, I have been in canada since 2017 and iam seeking for job in my field where I found its really difficult especially in engineering field, I join Hack Your Future because i want to learn a new thing related to Web and I found that its my interesting  </p>
+    </header>
+    <section class="goals">
+        <img src="https://www341.lunapic.com/editor/working/156501982863542728?7751299832" alt="dream" class="logo">
+    <h2>My Dream and Goals </h2>
+    <p>Since I have change my role in HYF I build up a new goal where i want to be qualified one program language, I have not yet decide which one until to go in depth in our course in order from me to indicate which one I prefer. My goal is to be a freelancer working in Web and making troubleshooting</p>
+    </section>
+    <div class="trip">
+    <img src="images/IMG_3886.JPG" alt="background" width="200">
```

Comment:
Instead of using inline CSS, you could create a rule in your external CSS to specify the width.